assets/src/styles/blocks/ENForm/components/_enform-full-width-bg.scss

```diff
@@ -1,5 +1,7 @@
+@import "../../master-theme/assets/src/scss/base/tokens";
+
 .enform-full-width-bg {
-  @include background-before-opacity($light-blue-bg);
+  @include background-before-opacity($beige-100);
```

Comment:
Why don't we use `var(--beige-100)` here? 🤔

Reply:
I think you should pass SASS vars instead of CSS vars